.babelrc

```diff
@@ -5,6 +5,7 @@
   ],
   "plugins": [
     "transform-object-rest-spread",
+    "transform-runtime",
```

Comment:
This is basically babel-polyfill for libraries. babel parses your code and look for any ES6 method calls, and inline those calls into the compiled bundle. This also removes all those _extend calls in the compiled code as well.

The only reason this is here is because I'm using Object.entries() for convenience.